pkg/controller/installation/core_controller.go

```diff
@@ -453,6 +453,24 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, err
 	}
 
+	var managerTLSSecret *corev1.Secret
+	managerTLSSecret, err = utils.ValidateCertPair(r.client,
+		render.ManagerTLSSecretName,
+		render.ManagerSecretKeyName,
+		render.ManagerSecretCertName,
+	)
+
+	if instance.Spec.ClusterManagementType == operator.ClusterManagementTypeManagement {
+		if managerTLSSecret == nil {
```

Comment:
This should be nil for a new cluster, so I don't think we should have this check.